_posts/2019-01-26-IndieBottlers.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: "Independent Bottlers"
+date: 2019-01-09 19:00:00
+tags: [events]
+share: y
+disqus: y
+image:
+    src: /images/2014-10/header.png
+    desc: Whisky Tasting - Independent Bottlers
+---
+
+We'll see you on **January 26, 2019 starting at 7PM** for our Independent Bottlers whisky tasting. This is a new one: independent bottlers offer a new take on existing brands and seek out the best anywhere. Come and see what WE found.
+
+Tickets to go on sale Wednesday, November 7th at 8AM.  The link shall follow shortly before that, via [email newsletter][3], [Slack][4], [Facebook][5], [Twitter][6], and at [whiskydev.com/tickets][1].
```

Comment:
Tickets go on sale when?

Reply:
And this is why we review.  Thanks for being so gaddamn keen Marc.